com.unity.netcode.gameobjects/Tests/Editor/Build/BuildTests.cs

```diff
@@ -7,6 +7,7 @@
 
 namespace Unity.Netcode.EditorTests
 {
+
```

Comment:
nit: only changes in this entire file are whitespace - maybe not worth including?